src/analyze/analyze.c

```diff
@@ -211,6 +229,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "  timespan SPAN...           Validate a time span\n"
                "  security [UNIT...]         Analyze security of unit\n"
                "  inspect-elf FILE...        Parse and print ELF package metadata\n"
+               "  memory [SERVICE NAME...]   Dump memory stats of a D-Bus service\n"
```

Comment:
as earlier, i'd call this "malloc", since it's about allocation, not about memory in general.

Reply:
and maybe call the argument [DBUS SERVICE…]